Classes/Emogrifier.php

```diff
@@ -530,11 +536,21 @@ private function fillStyleAttributesWithMergedStyles()
     private function generateStyleStringFromDeclarationsArrays(array $oldStyles, array $newStyles)
     {
         $combinedStyles = array_merge($oldStyles, $newStyles);
+
+        $cacheKey = serialize($combinedStyles);
```

Comment:
I have tested `json_encode` in the past for similar purposes; `serialize` is consistantly faster with less memory overhead. Don't know about `var_export`, though I doubt all the extra formatting would help.